src/lib.rs

```diff
@@ -356,6 +357,22 @@ impl FromStr for Ulid {
     }
 }
 
+impl TryFrom<&'_ str> for Ulid {
+    type Error = DecodeError;
+
+    fn try_from(value: &'_ str) -> Result<Self, Self::Error> {
+        Ulid::from_string(value)
+    }
+}
+
+impl TryFrom<String> for Ulid {
```

Comment:
Let's remove this impl. This shouldn't ever be used as there's no upside vs calling with `&str`.

Reply:
Right, that's fair. Wilco